models/model.js

```diff
@@ -82,5 +82,32 @@ exports.getArticles = async() => {
 
     catch(err) {
         
+    }
+}
+
+exports.getAllComments = async (article_id) => {
+    try {
+        const query = await connection.query(`
+
+        SELECT * FROM comments WHERE article_id = ${article_id};
+
+        `)
+
+        const {rows} = query
+
+        if (rows.length === 0) {
+            return Promise.reject({msg: 'no comments found for article ID'})
+        }
+
```

Comment:
If you add in a status code in your Promise.reject, then you can reuse this for future errors.